python-sdk/nuscenes/eval/predict/README.md

```diff
@@ -0,0 +1,73 @@
+# nuScenes predicton task
+In this document we present the rules, submission process, and evaluation metrics of the nuScenes prediction task.
+
+## Overview
+- [Introduction](#introduction)
+- [Participation](#participation)
+- [Challenges](#challenges)
+- [Submission rules](#submission-rules)
+- [Results format](#results-format)
+- [Evaluation metrics](#evaluation-metrics)
+
+## Introduction
+The goal of the nuScenes prediction task is to predict the future trajectories of objects in the nuScenes dataset.
+A trajectory is a sequence of x-y locations. For this challenge, the predictions are 6-seconds long and sampled at
+2 hertz.
```

Comment:
I think we originally mentioned including everyone (cars, cyclists, pedestrians) but it would be better to do just cars since that's the data we tested our baselines on.